src/css/hero.css

```diff
@@ -0,0 +1,316 @@
+.hero {
+    width: 320px;
+    height: 380px;
+    background-image:
+        linear-gradient(to left, rgba(17, 17, 17, 0) 10%, rgba(17, 17, 17, 1) 66%),
+        url("/img/image.png");
+    background-size: cover;
+    background-position: center;
+}
+
+.hero-text-cont {
+    width: 242px;
+    height: 286px;
+    padding-top: 40px;
+    margin-left: 20px;
+}
+
+.title {
+    width: 232px;
+    padding-bottom: 12px;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 37.5px;
+    text-align: left;
+}
+
+.star-rating {
+    display: flex;
+    gap: 4px;
+    padding-bottom: 20px;
+}
+
+.desc {
+    width: 177px;
+    height: 66px;
+    margin-bottom: 20px;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 16.41px;
+    text-align: left;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.watch-btn {
+    width: 117px;
+    height: 38px;
+    border-radius: 74px;
+    background: linear-gradient(141.22deg, #FFC226 9.4%, #F84119 91.91%);
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 14.06px;
+    text-align: center;
+    color: #111111;
+    position: relative;
+    transition: all .2s;
+}
+
```

Comment:
jak coś się powtarza to możesz to połączyć w jedno na zasadzie

.watch-btn,
.details-btn {  
    width: 117px;
    height: 38px;
   }
   
  a resztę dodajesz oddzielnie, jak jest dużo powtórzeń to dużo krótszy kod będzie